spec/factories/distributions.rb

```diff
@@ -35,6 +35,7 @@
       storage_location { create :storage_location, :with_items, item: item, organization: organization }
 
       after(:build) do |instance, evaluator|
+        # NOTE: does this need to run if item is passed in?
```

Comment:
The answer is yes, but danged if I can remember why. I remember specifically trying to shortcut this and it not working. :(